TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -337,7 +337,7 @@ protected function initializeUpdateAction()
     public function updateAction(Asset $asset)
     {
         $this->assetRepository->update($asset);
-        $this->addFlashMessage('assetHasBeenUpdated', '', Message::SEVERITY_OK, [htmlspecialchars($asset->getLabel())]);
+        $this->addFlashMessage('Asset "%s" has been updated.', '', Message::SEVERITY_OK, [htmlspecialchars($asset->getLabel())], 1469083898);
```

Comment:
`htmlspecialchars()` should not be required here (it's the task of the view to escape strings)